Code/GraphMol/Depictor/Wrap/testDepictor.py

```diff
@@ -616,90 +618,155 @@ def assertMolMatchesCoordMap(self, mol, coord_map):
+                32: Geometry.Point2D(1.5, 1.5),
+                33: Geometry.Point2D(0, 1.5)
+            }
+            rdDepictor.Compute2DCoords(two_linked_templates, coordMap=ring_system_coord_map, useRingTemplates=True)
+            self.assertMolMatchesCoordMap(two_linked_templates, ring_system_coord_map)
+            # atoms 10, 11, and 13 are in this template so the ring template should not be used
+            assert not self.molMatchesTemplate(two_linked_templates, template1)
+            assert self.molMatchesTemplate(two_linked_templates, template2)
+
+            # when a coord map contains a single atom, even if it is a part of a ring
+            # system, ring system templates should be used and the coord map should be
+            # followed
+            single_atom_coord_map = {
+                10: Geometry.Point2D(0, 0)
+            }
+            rdDepictor.Compute2DCoords(two_linked_templates, coordMap=single_atom_coord_map, useRingTemplates=True)
+            self.assertMolMatchesCoordMap(two_linked_templates, single_atom_coord_map)
+            assert self.molMatchesTemplate(two_linked_templates, template1)
+            assert self.molMatchesTemplate(two_linked_templates, template2)
+
+
+    def testSetRingSystemTemplates(self):
+        with rdDepictor.UsingCoordGen(False):
+            mol = Chem.MolFromSmiles("C1CC2CCOC3OC4CCC(C1)C23OO4")
+            default_template = Chem.MolFromSmiles("C1CC2CCOC3OC4CCC(C1)C23OO4 |(3.53,-1.22,;3.53,0.3,;2.21,1.06,;2.21,2.59,;0.89,3.35,;-0.43,2.59,;-0.43,1.06,;-1.9,0.65,;-2.47,-0.76,;-1.71,-2.08,;-0.2,-2.29,;0.89,-1.22,;2.21,-1.99,;0.89,0.3,;0.12,-0.83,;-1.19,-1.25,)|")
+            user_provided_template = Chem.MolFromSmiles("C1CC2CCOC3OC4CCC(C1)C23OO4 |(-0.5537,-3.1595,;-1.6057,-2.003,;-1.4262,-0.4072,;-2.9804,0.0271,;-3.5191,1.502,;-2.2028,2.3562,;-0.6818,1.8511,;1.0592,1.4391,;2.6123,1.8366,;3.5191,0.5341,;2.6067,-0.7521,;1.0061,-0.773,;0.7888,-2.3546,;-0.0405,0.5251,;0.4049,2.3,;1.7604,3.1594,)|")
+
+            # default templates are loaded automatically
+            rdDepictor.Compute2DCoords(mol, useRingTemplates=True)
+            assert self.molMatchesTemplate(mol, default_template)
+
+            # set to user-provided template, this will delete default templates
+            fpath = os.path.join(RDConfig.RDBaseDir, 'Code', 'GraphMol', 'Depictor', 'test_data', 'ring_system_templates.smi')
+            rdDepictor.SetRingSystemTemplates(fpath)
+            rdDepictor.Compute2DCoords(mol, useRingTemplates=True)
+            assert self.molMatchesTemplate(mol, user_provided_template)
+
+            # set back to default ring system templates
+            rdDepictor.LoadDefaultRingSystemTemplates()
+            rdDepictor.Compute2DCoords(mol, useRingTemplates=True)
+            assert self.molMatchesTemplate(mol, default_template)
+
+    def testSetBadRingSystemTemplates(self):
+        with tempfile.NamedTemporaryFile() as tmp_file:
```

Comment:
On Windows you need to include `delete=False` here or the call on l722 will always generate a `ValueError` because the file cannot be opened (so the tests do pass on windows, but for the wrong reasons).
```suggestion
        with tempfile.NamedTemporaryFile(delete=False) as tmp_file:
```
Please make this change for all of the uses of `NamedTemporaryFile()`

Reply:
@rachelnwalker I just did a PR against your branch with this block of suggested changes.
When those are made the CI builds should all pass.
Assuming that works, I think this is ready to go

Reply:
Thank you! I just committed these changes and added `delete=False` for each of these calls